.github/workflows/go.yml

```diff
@@ -28,5 +28,5 @@ jobs:
 
       - name: Test
         run: |
-          ginkgo -r -cover -coverprofile=coverage.txt -covermode=atomic
+          ginkgo -r -cover -coverprofile=coverage.txt -covermode=atomic -- --rode-api-host="localhost:50051"
```

Comment:
What's the reason for passing in the `rode-api-host` flag here? Seems to have broken the test, the Action has been running for almost 40 minutes now

Reply:
It was broken before adding the flag here. I'm trying to figure out why it runs locally but not in the pipeline